rss2email/feed.py

```diff
@@ -562,7 +565,17 @@ def _get_entry_date(self, entry):
                 if entry.get(kind, None):
                     datetime = entry[kind]
                     break
-        return _time.strftime("%a, %d %b %Y %H:%M:%S -0000", datetime)
+        LOCALE_LOCK = _threading.Lock()
+        @_contextlib.contextmanager
+        def setlocale(name):
+            with LOCALE_LOCK:
+                saved = _locale.setlocale(_locale.LC_ALL)
+                try:
+                    yield _locale.setlocale(_locale.LC_ALL, name)
+                finally:
+                    _locale.setlocale(_locale.LC_ALL, saved)
+        with setlocale("C"):
+            return _time.strftime("%a, %d %b %Y %H:%M:%S -0000", datetime)
```

Comment:
After a bit of research, I found [email.utils.formatdate](https://docs.python.org/3/library/email.utils.html#email.utils.formatdate), which gives you a compliant header value with much less effort, so this line would simply change to:
```python
…
from email.utils import formatdate as _formatdate
…
        …
        _formatdate(_time.mktime(datetime))
…
```